### ISSUE TITLE
Insufficient colour contrast of select box items in Windows 10

### ISSUE THREAD

**Issue by kr8n3r** [alphagov/reported-bugs#17]
**Upstream bug:** https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/11826078/
**Date:** 2018-04-28
**Reported by:** Jani Kraner

---

### Overview

Select box - colour contrast is only 2.9:1 between the highlighted (white) text and pale blue background - this is in Windows 10 and checked with both IE and Firefox

Example select box where this happens: https://design-system.service.gov.uk/components/select/default/index.html

This is the default look of the Windows IE/Edge select boxes and not part of GOV.UK Frontend.

### Expected result

Darker background colour would need to be implemented by vendor

### Actual result

Closed as won't fix. We've done a fix in our codebase here: https://github.com/alphagov/govuk-frontend/blob/master/src/components/select/_select.scss#L23-L25

Response: 

> Hello,
> 
> Thank you for providing this update. If you need better accessibility to the browser elements, please consider using Windows high-contrast settings.
> 
> https://support.microsoft.com/en-us/help/13862/windows-use-high-contrast-mode
> 
> Best Wishes,
> The MS Edge Team
